tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -0,0 +1,186 @@
+<?php
+namespace Step\Acceptance\Administrator;
+
+class Content extends \AcceptanceTester
+{
+	/**
+	 * @Given There is a Add Content link
+	 */
+	public function thereIsAAddContentLink()
+	{
+		$I = $this;
+		$I->amOnPage('administrator/index.php?option=com_content&view=articles');
+		$I->clickToolbarButton('New');
+	}
+
+	/**
+	 * @When I fill mandatory fields for creating article
+	 */
+	public function iFillMandatoryFieldsForCreatingArticle(\Behat\Gherkin\Node\TableNode $fields)
+	{
+
+		$I = $this;
+		// iterate over all rows
+		foreach ($fields->getRows() as $index => $row) {
+			if ($index === 0) { // first row to define fields
+				$keys = $row;
+				continue;
+			}
+			else
+			{
+				if ($row[0] == "title")
```

Comment:
Can you take a look at the comments i made at Puneets PR? There are several issues here, starting from code style to readability ;)